components/skus/browser/rs/cxx/src/lib.rs

```diff
@@ -164,6 +164,12 @@ mod ffi {
             order_id: String,
             receipt: String,
         );
+        fn submit_receipt_new_order(
```

Comment:
Here and below, we discussed that a more descriptive name would be `create_order_from_receipt`.

Reply:
addressed: https://github.com/brave/brave-core/pull/21773/commits/8e295eddc53eec6dc0a649158ee3ecd8ba8df20e